.circleci/config.yml

```diff
@@ -105,22 +129,54 @@ jobs:
               "${ARNOLD_IMAGE}" \
               ansible --version
 
+  # FIXME: we have deactivated plugins test coverage as the container user is
+  # not allowed to write to /app/.coverage and CircleCI does not allowed to
```

Comment:
s/allowed/allow

Reply:
:thinking: it's the container user that is not **allowed** to

Reply:
I nearly apologized... then I realized that there are 2 "allowed" in this sentence ^^

Reply:
Oops. you're right.